package.json

```diff
@@ -42,8 +42,8 @@
     "expo-updates": "~0.25.27",
     "expo-web-browser": "~13.0.3",
     "graphql": "^16.9.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.3.1",
```

Comment:
Is this necessary? I don't think React `18.3.1` is stable and this will conflict with `yarn expo install --fix`.